srcpkgs/python3-typed-ast/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-typed-ast'
+pkgname=python3-typed-ast
+version=1.3.1
+revision=1
+wrksrc="typed-ast-${version}"
+build_style=python3-module
+pycompile_module="typed_ast"
+hostmakedepends="python3-devel python3-setuptools"
```

Comment:
python3-devel should be in makedepends as you are linking against the python library.

Packages that have libraries you are linking against go into makedepends.

Please read up on the Manual: https://github.com/void-linux/void-packages/blob/master/Manual.md#explain_depends